website/docs/reference/resource-properties/deprecation_date.md

```diff
@@ -28,5 +28,46 @@ models:
 
 ## Definition
 
-The deprecation date of the model in YAML DateTime format. 
+The deprecation date of the model is formatted as a date, optionally with a timezone offset. Supported RFC 3339 formats include:
+- `YYYY-MM-DD hh:mm:ss.sss±hh:mm`
+- `YYYY-MM-DD hh:mm:ss.sss`
+- `YYYY-MM-DD`
 
+When `deprecation_date` does not include an offset from UTC, then it is interpreted as being in the system time zone of the dbt execution environment.
+
+## Explanation
+
+### Purpose
+
+Declaring a `deprecation_date` for a dbt model provides a mechanism to communicate plans and timelines for long-term support and maintenance and to facilitate change management.
+
+Setting a `deprecation_date` works well in conjunction with other [model governance](/docs/collaborate/govern/about-model-governance) features like [model versions](/docs/collaborate/govern/model-versions), but can also be used independently from them.
+
+### Warning messages
+
+When a project references a model that's slated for deprecation or the deprecation date has passed, a warning is generated. If it's a versioned model, with a newer version available, then the warning says so. This added bit of cross-team communication, from producers to consumers, is an advantage of using dbt's built-in functionality around model versions to facilitate migrations.
```

Comment:
👍 love this

Could we include a code snippet displaying the terminal/IDE output for one of these warnings? I think `UpcomingDeprecationReference` is the most compelling

Reply:
Addressed in 5ec8a46fbe98c253995f3b878ca158f296963e21